tests/fullydistributed_grids/copy_serial_tria_04.mpirun=7.output

```diff
@@ -3,158 +3,158 @@ DEAL:0:1d::n_levels:                  1
 DEAL:0:1d::n_cells:                   20
 DEAL:0:1d::n_active_cells:            20
 DEAL:0:1d::
-DEAL:0:1d::n_dofs:                             42
-DEAL:0:1d::n_locally_owned_dofs:               42
+DEAL:0:1d::n_dofs:                             257
+DEAL:0:1d::n_locally_owned_dofs:               37
 DEAL:0:1d::
 DEAL:0:2d::n_levels:                  1
 DEAL:0:2d::n_cells:                   2502
 DEAL:0:2d::n_active_cells:            2502
 DEAL:0:2d::
-DEAL:0:2d::n_dofs:                             10338
-DEAL:0:2d::n_locally_owned_dofs:               10338
+DEAL:0:2d::n_dofs:                             66049
+DEAL:0:2d::n_locally_owned_dofs:               9585
 DEAL:0:2d::
 DEAL:0:2d::n_levels:                  1
 DEAL:0:2d::n_cells:                   5968
 DEAL:0:2d::n_active_cells:            5968
 DEAL:0:2d::
-DEAL:0:2d::n_dofs:                             53441
-DEAL:0:2d::n_locally_owned_dofs:               53441
+DEAL:0:2d::n_dofs:                             274625
+DEAL:0:2d::n_locally_owned_dofs:               41309
 DEAL:0:2d::
 
 DEAL:1:1d::n_levels:                  1
 DEAL:1:1d::n_cells:                   20
 DEAL:1:1d::n_active_cells:            20
 DEAL:1:1d::
-DEAL:1:1d::n_dofs:                             41
-DEAL:1:1d::n_locally_owned_dofs:               41
+DEAL:1:1d::n_dofs:                             257
+DEAL:1:1d::n_locally_owned_dofs:               36
 DEAL:1:1d::
 DEAL:1:2d::n_levels:                  1
 DEAL:1:2d::n_cells:                   2618
 DEAL:1:2d::n_active_cells:            2618
 DEAL:1:2d::
-DEAL:1:2d::n_dofs:                             10829
-DEAL:1:2d::n_locally_owned_dofs:               10829
+DEAL:1:2d::n_dofs:                             66049
+DEAL:1:2d::n_locally_owned_dofs:               9456
 DEAL:1:2d::
 DEAL:1:2d::n_levels:                  1
 DEAL:1:2d::n_cells:                   6556
 DEAL:1:2d::n_active_cells:            6556
 DEAL:1:2d::
-DEAL:1:2d::n_dofs:                             59555
-DEAL:1:2d::n_locally_owned_dofs:               59555
+DEAL:1:2d::n_dofs:                             274625
```

Comment:
Without looking closely, this seems to be a rather big size for a test. Is this size necessary or is it enough with one level less?

Reply:
I have reduced the number of levels in some of the tests.